VTU/Sem3/DS_Lab/04Queue/04Queue.md

```diff
@@ -1,21 +1,21 @@
-##PROGRAM 4 : queue.c
-##Aim:
-##Design, develop, and execute a program in C to simulate the working of a queue of integers using an array. Provide the following operations:
+## PROGRAM 4 : queue.c
+## Aim:
+## Design, develop, and execute a program in C to simulate the working of a queue of integers using an array. Provide the following operations:
 ##1.Insert
```

Comment:
Do we not want this part?

Reply:
@cyberhck What are you saying?

Reply:
`##1.Insert` part.

Reply:
Hi guys. Any update on

Reply:
I asked weather or not `##1.Insert` is supposed to be changed too?

Reply:
Yeah. Should be bullets. But this pull is not meant for those intricate changes. Its just a fix for the Headers anywhere thats present without space. I think next commit can handle such issues.